test/e2e/cluster_extension_admission_test.go

```diff
@@ -64,12 +66,33 @@ func TestClusterExtensionPackageUniqueness(t *testing.T) {
 	defer deleteClusterExtension(clusterExtension2)
 
 	t.Log("update second resource with package which already exists on the cluster")
-	clusterExtension2.Spec.PackageName = firstResourcePackageName
-	err = c.Update(ctx, clusterExtension2)
+	intent := &ocv1alpha1.ClusterExtension{
+		TypeMeta: metav1.TypeMeta{
+			APIVersion: ocv1alpha1.GroupVersion.String(),
+			Kind:       "ClusterExtension",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name: clusterExtension2.Name,
```

Comment:
what should be the value of `clusterExtension2.Name` at this point? I think it might be "" because in its definition above uses `GenerateName` (or am I missing something?)

Reply:
After `c.Create(ctx, clusterExtension2)` we would have `Name` populated with the real one from the server.

Reply:
Right! I was thinking of `c.Create` as non-mutating. But, of course, it makes sense to update the input, since the api server will anyway return the resource on create.